docker-compose.yml

```diff
@@ -280,7 +280,7 @@ services:
       - "logging=true"
   lod-sbmb:
     # Trigger needs to be configured in server cron
-    # cd /data/app-kaleidos && /usr/local/bin/docker-compose run lod-sbmb -g -s $(date +\%Y) -e $(date +\%Y) -n decreet -f decret -t DECREE -o /data -c /data/cache -w 10 && mv ./data/staatsblad/decreet* ./data/staatsblad/nl && mv ./data/staatsblad/decret* ./data/staatsblad/fr
+    # cd /data/app-kaleidos && /usr/local/bin/docker-compose run lod-sbmb -g -s $(date +\%Y) -e $(date +\%Y) -n reet -f decret -t DECREE -o /data -c /data/cache -w 10 && mv ./data/staatsblad/decreet* ./data/staatsblad/nl && mv ./data/staatsblad/decret* ./data/staatsblad/fr
```

Comment:
Not sure why this change was made, and which version is meant to be kept. To be reviewed on Monday